diff -urN 54321/archs/Linux/variables.GNU 54321-good/archs/Linux/variables.GNU
--- 54321/archs/Linux/variables.GNU	Thu Nov  8 16:54:01 2001
+++ 54321-good/archs/Linux/variables.GNU	Sat May 11 14:07:36 2002
@@ -5,8 +5,8 @@
 MKDIR_CMD = test -d $(@D) || mkdir -p $(@D)
 ############################################################
 
-SDL := /usr/local
-SDL_IMAGE := /usr/local
+SDL := /usr/X11R6
+SDL_IMAGE := /usr/X11R6
 
 CPPFLAGS += -I${SDL}/include/SDL -I${SDL_IMAGE}/include/SDL
 LDFLAGS += \
